IPs/RSKIP213.md

```diff
@@ -0,0 +1,77 @@
+# Simple Transaction Versioning System
+
+
+|RSKIP          | 213 |
+| :------------ |:-------------|
+|**Title**      |Simple Transaction Versioning System|
+|**Created**    |FEB-2021 |
+|**Author**     |SDL|
+|**Purpose**    |Sca |
+|**Layer**      |Core |
+|**Complexity** |1|
+|**Status**     |Draft |
+
+
+# **Abstract**
+
+This RSKIP proposes the addition of a version field so that transaction parsing can start after transaction version has been verified. This reduces the amount of changes required for the node and software wallets.
+
+## Motivation
+
+Many features, like ephemeral calldata, multi-signed transactions and LTCP transaction compression require adding more information to transactions. However currently RSK transactions do not have a way to indicate a new transaction format is being used. We propose a transaction format that embeds a version field in the current nonce position, keeping high compatibility with wallets.
```

Comment:
Many features, like ephemeral calldata, multi-signed transactions and LTCP transaction compression require adding more information to transactions. However currently RSK transactions do not have a way to indicate that a new transaction format is being used. We propose a transaction format that embeds a version field in the current nonce position, keeping high compatibility with wallets.

Reply:
same here

Reply:
The word that is added into "Many features, like ephemeral calldata, multi-signed transactions and LTCP transaction compression require adding more information to transactions. However currently RSK transactions do not have a way to indicate **that** a new transaction format is being used. We propose a transaction format that embeds a version field in the current nonce position, keeping high compatibility with wallets."